company-profiles/lambert-labs.md

```diff
@@ -0,0 +1,29 @@
+# Lambert Labs
+
+## Company blurb
+
+We are a Python-focused development agency that builds software for a variety of clients who work in diverse sectors such as Technology, Education, Legal and Finance.
+
+## Company size
+
+0-20 (as of June 2021)
+
+## Remote status
+
+We have an office in Earls Court, London, but some of the team work remotely.
```

Comment:
Can you elaborate a bit more about how you work with those that are remote if some of the company are in offices?

Reply:
Thanks Doug, I've elaborated here